_pages/en/esdc-it-strategy.md

```diff
@@ -52,43 +45,114 @@ But a complex and uncertain world rewards an empirical cycle of trying, observin
+
+The above short term outcomes are expected to lead to the following medium term ones.
+
+<!--markdownlint-disable MD029 MD036-->
+**1) Business Problem solving includes technology consideration**
+
+Lines of business start engaging more and more IT teams to help them _solve business problems_. This involves:
+
+- Moving away from business requirements, and towards clarity of the problem statements instead
+- Engaging IT teams earlier in the process, fostering a collaboration of problem solving
+- Experimenting more with proof of concepts or pilots before starting a project
+- Committing subject matter experts, including policy makers, during the project execution phase
+
+**2) IITB is becoming a Generative type organization**
+
+An organization culture that is high-trust and emphasizes information flow between teams is necessary to achieve excellence. The [Westrum organization topology model](https://cloud.google.com/solutions/devops/devops-culture-westrum-organizational-culture), by Dr. Ron Westrum, will be used as a guide to move towards and measure the organizational culture as follows:
+
+- The cooperation level between teams
+- How the reception of bad news is treated
+- The level at which shared responsibilities are encouraged and observed
+- The level at which bridging, breaking down silos, is encouraged and observed
+- How the response to failure is managed
+- The level at which experimentation is encouraged and observed
 
-The approach to the IT Strategy is **not** to complete each of the diagram's block before publishing them as a single plan.
-Instead, the ESDC IT Strategy team will work with the IT community to continuously diagnose key organizational problems, waste, lead times and governance structures that inhibit the desired changes in behaviours.
+**3) Reduced gap between the user and IT**
+
+Moving towards an empirical cycle of trying, observing, and correcting when managing IT initiatives will require involving the user throughout the exercise. This involves:
+
+- Increased focus on outcomes over advance planning activities
+- Increasing the frequency of deployment in production to receive feedback from end-users
+- Allowing IT teams to engage directly with end-users
+- Increased awareness of policy makers on the impacts of policies to end-users and IT systems
+
+<!--markdownlint-enable MD029 MD036-->
+
+### Long Term Outcomes (Impact)
+
+The long term impacts of the IT Strategy is to create the following two conditions:
+
+<!--markdownlint-disable MD029 MD036-->
+**1) Technology is a shared accountability between stakeholders**
+
+Lines of businesses _share the costs and risks_ of using technology with the IT branch.
+Before an IT-enabled initiative is launched, the business problem to solve with technology _is clear_, business experts _fully participate_ in the execution of the initiative in partnership with their technology colleagues, and _rapid feedback_ from end-users is not only promoted, it is _enabled_.
+
```

Comment:
Perhaps add "dismantling of business vs IT paradigm", obviously my wording is poor but you get the idea

Reply:
this is good. I'll word it as "the business vs IT paradigm is replaced with team partnership paradigm"